geode-docs/topologies_and_comm/cs_configuration/setting_up_a_client_server_system.html.md.erb

```diff
@@ -53,15 +53,28 @@ Configure your server and client processes and data regions to run your client/s
     
         <client-cache>
            <pool name="publisher" subscription-enabled="true">
-              <locator host="lucy" port="41111"/> 
-              <locator host="lucy" port="41111"/> 
+              <locator host="lucy1" port="41111"/>
+              <locator host="lucy2" port="41111"/>
            </pool>
            ...
            <region name="clientRegion" ...
               <region-attributes pool-name="publisher" ...
 
     You do not need to provide the complete list of locators to the clients at startup, but you should provide as complete a list as possible. The locators maintain a dynamic list of locators and servers and provide the information to the clients as needed.
 
+    When TLS (SSL) is used clients can also be directed to go through a SNI gateway such as Istio or HAProxy to reach locators and servers.  To do this add the following to your cache.xml pool configuration:
+        <pool... >
+          <socket-factory>
+            <class-name>org.apache.geode.cache.client.proxy.SniSocketFactory</class-name>
+            <parameter name="hostname">
+              <string>my-gateway-address</string>
+            </parameter>
+            <parameter name="port">
+              <string>my-gateway-port-number</string>
```

Comment:
for consistency with the other .erb file, should we have a number here?

Reply:
12345